source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst

```diff
@@ -0,0 +1,50 @@
+.. rn:: 1.0.0
+
+Percona Kubernetes Operator for XtraDB Cluster
+==============================================
+
+Percona announces the general availability of |Percona Kubernetes Operator for XtraDB Cluster| 1.0.0 on May 24, 2019. This release is now the current GA release in the 1.0 series. Download the latest version from the Percona Software Repositories. Please see the `GA release announcement`. All of Percona's software is open-source and free.
+
+The Percona Kubernetes Operator for Percona XtraDB Cluster automates the creation, modification, and deletion of members of your Percona XtraDB Cluster environment. The Operator can be used to instantiate a Percona XtraDB Cluster, or scale an existing Cluster.
+
+The Operator contains the necessary Kubernetes settings and provides a consistent Percona XtraDB Cluster instance. The Percona Kubernetes Operators are based on best practices for configuration and setup of the Percona XtraDB Cluster.
```

Comment:
The operator doesn't contain settings :(

Reply:
Would "The Operator provides a consistent Percona XtraDB Cluster instance." be more accurate?

Reply:
not an option
the next sentence also starts very similar "The Kubernetes Operators provide a consistent way to package..."